.github/workflows/holochain-build-and-test.yml

```diff
@@ -105,16 +105,22 @@ jobs:
 
           git status
 
-          # if a nix store path was passed, ensure it exists locally
+          # If a nix store path was passed, ensure it exists locally. This will fetch the path from cachix.
           if [[ ${{ inputs.repo_path }} == "/nix/"* ]]; then
             nix-store --realise ${{inputs.repo_path }}
+            
+            # Temporary workaround for https://github.com/NixOS/nix/issues/10202
+            git config --global --add safe.directory ${{ inputs.repo_path }}
```

Comment:
Asks Git to accept that the current user and the owner of the repo fetched from Cachix are different. See the next comment